net/instaweb/rewriter/public/rewrite_options.h

```diff
@@ -4220,6 +4247,8 @@ class RewriteOptions {
   int experiment_percent_;  // Total traffic going through experiments.
   std::vector<ExperimentSpec*> experiment_specs_;
 
+  // Headers to add resource responses.
+  std::vector<NameValue*> resource_headers_;
```

Comment:
Yes, doing this in a follow-up pull-request makes sense.  Please leave a TODO indicating your intention.  Thanks!!